tests/helpers/mod.rs

```diff
@@ -246,26 +256,28 @@ fn parse_spec(text: String) -> Spec {
   files.push(current_file.unwrap());
   let output_file =
     files.remove(files.iter().position(|f| f.specifier == "output").unwrap());
-  let diagnostics = take_vec_file(&mut files, "diagnostics");
-  let workspace_members = take_vec_file(&mut files, "workspace_members");
+  let diagnostics = take_file(&mut files, "diagnostics");
+  let workspace_members = take_file(&mut files, "workspace_members");
+  let lockfile_jsr_packages = take_file(&mut files, "lockfile_jsr_packages");
```

Comment:
Nice! I think you still have the test that uses this locally though? Or I might be missing it.